br/pkg/lightning/backend/local/local.go

```diff
@@ -1531,19 +1531,18 @@ loopWrite:
 					continue
 				}
 			}
+			if err != nil {
```

Comment:
after I comment this part, the UT still passed

```diff
diff --git a/br/pkg/lightning/backend/local/local.go b/br/pkg/lightning/backend/local/local.go
index c21d363c90..24ccf23588 100644
--- a/br/pkg/lightning/backend/local/local.go
+++ b/br/pkg/lightning/backend/local/local.go
@@ -1531,11 +1531,11 @@ loopWrite:
                                        continue
                                }
                        }
-                       if err != nil {
-                               log.FromContext(ctx).Warn("batch ingest fail after retry, will retry import full range", log.ShortError(err),
-                                       logutil.Region(region.Region), zap.Reflect("meta", ingestMetas))
-                               return errors.Trace(err)
-                       }
+                       //if err != nil {
+                       //      log.FromContext(ctx).Warn("batch ingest fail after retry, will retry import full range", log.ShortError(err),
+                       //              logutil.Region(region.Region), zap.Reflect("meta", ingestMetas))
+                       //      return errors.Trace(err)
+                       //}
                }
 
                engine.importedKVSize.Add(rangeStats.totalBytes)
diff --git a/br/pkg/lightning/backend/local/local_test.go b/br/pkg/lightning/backend/local/local_test.go
index 32cc72441c..b032c8bad3 100644
--- a/br/pkg/lightning/backend/local/local_test.go
+++ b/br/pkg/lightning/backend/local/local_test.go
@@ -1499,5 +1499,4 @@ func TestIngestFailOnFirstBatch(t *testing.T) {
        }, []byte{}, []byte("b"), 0, 0)
        require.ErrorContains(t, err, "Suspended { time_to_lease_expire: 1.204s }")
        require.Equal(t, []uint64{1}, apiInvokeRecorder["Write"])
-       require.Equal(t, []uint64{1, 1, 1, 1, 1}, apiInvokeRecorder["Ingest"])
 }
```

`apiInvokeRecorder["Ingest"]` is `1` repeated 15 times so I also remove its check

Reply:
the check is required, it should repeat 5 times on the first batch, and report err, not continue

Reply:
`        require.ErrorContains(t, err, "Suspended { time_to_lease_expire: 1.204s }")` this check is not deleted and it passed

Reply:
cause this: https://github.com/pingcap/tidb/pull/50282/files#diff-2c33db76e1a37e0c61c9a3af56d0b96174e83cbf622672e9330e013cf9f10d66R1458

we could change it to 5, but it didn't change the correctness of the ut